react-native-webview.podspec

```diff
@@ -2,7 +2,6 @@ require "json"
 
 package = JSON.parse(File.read(File.join(__dir__, "package.json")))
 
-folly_version = '2021.06.28.00-v2'
```

Comment:
By removing this, we rely on the version provided by React Native. Given that the lib does not uses folly directly, it will improve the compatibility. Otherwise, any time we bump folly in React Native, we have to bump the lib as well.